src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Utils;
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Core.Parser
+{
+    internal class ParseFormulasResult
+    {
+        internal Dictionary<DName, ParseResult> NamedFormulas { get; }
```

Comment:
I still feel like this should be returning <DName, string>, and then be fed back into the main parser as a 2nd step from the host

Reply:
Can you elaborate on that?  That will force double parsing. 

Unless there's a really compelling reason, I think we should use TexlNode, simiar to how ParseRecord does it. 

(I'm not sure whether DName or Identifier is the right key type here)

(@bryancroteau-MSFT , fyi)

Reply:
Yeah, actually, that's fair. The division can be handled after this.